tools/Az.Tools.Predictor/Az.Tools.Predictor/AzPredictor.cs

```diff
@@ -43,40 +52,74 @@ internal sealed class AzPredictor : ICommandPredictor, IDisposable
         /// <inhericdoc />
         public Guid Id => Identifier;
 
-        /// <inhericdoc />
-        public bool SupportEarlyProcessing => true;
-
-        /// <inhericdoc />
-        public bool AcceptFeedback => true;
-
         internal static readonly Guid Identifier = new Guid("599d1760-4ee1-4ed2-806e-f2a1b1a0ba4d");
 
         private const int SuggestionCountForTelemetry = 5;
 
-        private static readonly string[] CommonParameters = new string[] { "location" };
-
-        private readonly IAzPredictorService _service;
-        private readonly ITelemetryClient _telemetryClient;
-        private readonly Settings _settings;
-        private readonly IAzContext _azContext;
+        private static readonly HashSet<string> _azAccountCommands = new HashSet<string>(new string[]
+                {
+                    "Connect-AzAccount", "Clear-AzContext", "Disconnect-AzAccount", "Import-AzContext", "Remove-AzContext", "Set-AzContext"
+                },
+                StringComparer.InvariantCultureIgnoreCase);
+        private static readonly PredictiveSuggestion _surveySuggestion = new PredictiveSuggestion("Open-AzPredictorSurvey # Would you like to provide us feedback?");
```

Comment:
@dcaro, What do you think about this message (in comment) showing in the suggestion list?

Reply:
I suggest: "Open-AzPredictorSurvey # Run this command and tell us about your experience with Az Predictor"